roles/pulp_common/vars/main.yml

```diff
@@ -11,7 +11,7 @@ pulp_install_plugins_normalized_yml: |-
 # pip/PyPI only uses dashes, not underscores.
 pulp_install_plugins_normalized: "{{ pulp_install_plugins_normalized_yml | from_yaml }}"
 pulp_certs_dir: "{{ pulp_config_dir }}/certs"
-# Users should not set this variable, instead using `pulp_settings`
+# Users should not set these variables, but instead use `pulp_settings`
 __pulp_common_pulp_settings_defaults:
```

Comment:
I'm confused by this line actually. I expected the installer to only define settings defaults for "required settings" which Pulp itself cannot set and requires a user to set. For example, `SECRET_KEY` must be set https://docs.pulpproject.org/pulpcore/settings.html#secret-key

If the same settings are set here as in [pulpcore.app.settings](https://github.com/pulp/pulpcore/blob/master/pulpcore/app/settings.py#L174-L181) I think we should remove them from here.

Reply:
This concern is not really related to this PR btw ... We don't need to stop the merge due to this.

Reply:
1. For settings that are not required by Pulp, we often set them for the convenience of users. (We want pulp / its installer to "just work" out of the box.) For example, `token_server` is set to the ansible detected FQDN, and to https (https is enabled via the webserver role). pulpcore could called `socket.getfqdn()`, but wouldn't know which protocol.
2. For settings that are required by Pulp, and are security sensitive, we will not set them. We tell users to always specify `SECRET_KEY`.
3. For settings that are require by Pulp, and are not security sensitive, we set them in the installer because the values are based in the installer. content_origin is currently set in [the example playbook's vars](https://github.com/pulp/pulp_installer/blob/master/playbooks/example-use/group_vars/all#L18), instead of this data structure, to `https://{{ ansible_fqdn }}`. Again, https is an aspect of the installer. That said, it seems like a bad user experience of the installer to have similar variables defaulted in 2 different places.